src/entities/LiveLoopEntity.ts

```diff
@@ -0,0 +1,67 @@
+import { Entity } from './entity';
+import { World } from 'src/world';
+import { Subscription } from 'rxjs';
+import { controlEvents, eventIs } from 'src/controls';
+import THREE = require('three');
+import { LiveLoopName } from 'src/generation/directory';
+import LiveLoop from 'src/generation/LiveLoop';
+
+export interface LiveLoopEntityDefinition {
+  name: LiveLoopName;
+  mesh: THREE.Mesh;
+}
+
+/**
+ * A LiveLoopEntity is made when a LiveLoopTemplate is selected in the world.
+ * The LiveLoopEntity actually plays the live loop.
+ */
+export default class LiveLoopEntity implements Entity {
+  mesh: THREE.Mesh;
+  liveloop: LiveLoop;
+  subscription: Subscription;
+  selected: boolean = false;
+  amplitude: number = 0.5;
+
+  constructor(definition: LiveLoopEntityDefinition) {
+    this.liveloop = new LiveLoop(definition.name);
```

Comment:
I believe so? So the entity is created when it is dropped in the world (after selecting template), and that's when we want the music to start playing, right?